pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -539,6 +540,122 @@ func TestWithBinding(t *testing.T) {
 	}
 }
 
+func TestIsSchedulableAfterPersistentVolumeChange(t *testing.T) {
+	pvLister := tf.PersistentVolumeLister{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "Vol_2", Labels: map[string]string{v1.LabelFailureDomainBetaZone: "us-west1-b"}},
+		},
+	}
+
+	pvcLister := tf.PersistentVolumeClaimLister{
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "PVC_1", Namespace: "default"},
+			Spec:       v1.PersistentVolumeClaimSpec{VolumeName: "Vol_1"},
+		},
+	}
```

Comment:
Do we need them?

Reply:
It is not needed, I've removed it. Thanks!